unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts

```diff
@@ -133,16 +135,15 @@ export class DgraphDataSource extends RESTDataSource {
     const categoryString = this.getEntityCategoryString(categories);
     const orderString = this.getEntitiesOrderString(orderField, orderDesc);
     const filterString = this.getEntitiesFilterString(description, match);
-    const isMsupply = types.includes(EEntityType.UNIT_OF_USE);
 
+    // Required for backwards compatiblility with broken mSupply query. 
+    // Query incorrectly specifies medical_product as a type instead of category.
+    const isMsupply = types.includes(EEntityCategory.MEDICINAL_PRODUCT);
+
+    // TODO: migrate query to mSupply and remove hardcoded query mapping.
```

Comment:
Looking forward to it heh heh. When we update would be good to document which version(s) have the old one so we can guesstimate when to remove it (if ever? 😿 )